test/puppetlabs/trapperkeeper/authorization/rules_test.clj

```diff
@@ -68,6 +52,31 @@
       (testing (str "matching " x)
         (is (= (:rule (rules/match? rule (request "/path/to/resource" x))) rule))))))
 
+(deftest test-matching-query-parameters
```

Comment:
I think it would be good to have at least a few tests for equality where you have multiple query param keys, not just multiple values for one key, in the rule.